src/std/fs.ab

```diff
@@ -54,9 +54,9 @@ pub fun dir_create(path: Text): Null {
     }
 }
 
-/// Sets a file as executable.
+/// Changes the permission bits of a file.
 ///
-/// If the file doesn't exist, it returns a boolean and prints a message.
+/// If the file doesn't exist, it returns `false` and prints a message.
```

Comment:
To be honest I think that functions returning a `Bool` value type should be rewritten to failable equivalents if the returned `Bool` means whether or not the function succeeded.

Essentially the use of `Bool` should always indicate existence of certain attribute. Like for checking if something is switched on or off or ex. if a value exists in some array. In case of returning a success status, we should always return failure which gives more information about the error (different errors may have different exit codes).

But that's out of scope of this PR. I'll create an issue